docs/monitoring/metrics.md

```diff
@@ -699,6 +699,39 @@ Flink metric types are mapped to Prometheus metric types as follows:
 
 All Flink metrics variables (see [List of all Variables](#list-of-all-variables)) are exported to Prometheus as labels. 
 
+### PrometheusPushGateway (org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporter)
```

Comment:
Please add a section highlighting the differences and use-cases compared to the existing reporter.

In particular we should mention that this reporter, like the existing reporter, is not suited for short-lived jobs.

Reply:
thanks for review, I will improve the doc